app/models/about_user.rb

```diff
@@ -0,0 +1,11 @@
+class AboutUser < ActiveRecord::Base
```

Comment:
This file should have the model annotations in... check the other model files.

The gem is https://github.com/ctran/annotate_models, see if you can make it work?

Reply:
Yes, I updated the annotations for other models as well